audio_service/android/src/main/java/com/ryanheise/audioservice/AudioServicePlugin.java

```diff
@@ -945,7 +945,7 @@ private static RatingCompat raw2rating(Map<String, Object> raw) {
 
     private static String metadataToString(MediaMetadataCompat mediaMetadata, String key) {
         CharSequence value = mediaMetadata.getText(key);
-        if (value != null && value.length() > 0)
+        if (value != null)
```

Comment:
I vaguely remember there being a purpose to `value.length() > 0`. Maybe Android was filling in missing metadata with empty strings whenever we send a media item through the media API and it comes out the other end. But we no longer communicate between isolates by going through the media layer, we do it completely with dart-to-dart communication in the same isolate. So I'm happy to try this and see how it goes.